CHANGELOG.mkd

```diff
@@ -1,6 +1,13 @@
 CHANGELOG
 =========
 
+3.0.2
```

Comment:
It looks like we shipped 3.0.2 w/ some just some test removals?

Reply:
Yeah just saw that. Updated.